sdk/servicebus/service-bus/samples/typescript/src/atomManagementApi.ts

```diff
@@ -50,7 +53,7 @@ async function main(): Promise<void> {
     console.log(`Queue #${i + 1} - ${listQueuesResult[i].queueName}`);
   }
 
-  await serviceBusAtomManagementClient.deleteQueue("anotherTestQueue");
+  await serviceBusAtomManagementClient.deleteQueue("ranotherTestQueue");
```

Comment:
I think you've gone too 'r' crazy. I mean, you're adding 'r's' left and right here.

The power to 'r' corrupts absolutely.

Reply:
r